src/main/java/com/yetanalytics/xapi/model/LangTag.java

```diff
@@ -0,0 +1,87 @@
+package com.yetanalytics.xapi.model;
+
+import java.util.IllformedLocaleException;
+import java.util.Locale;
+import java.util.Locale.Builder;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Class representation of <a href="https://datatracker.ietf.org/doc/html/rfc5646">RFC 5646 Language Tags</a>
+ * allowing for retrieval of both the original String and the corresponding Locale.
+ */
+@JsonInclude(Include.NON_NULL)
+public class LangTag {
+    private final String languageTagString;
+    private final Locale languageTagLocale;
+
+    /**
+     * This constructor takes a RFC 5646 formatted String and converts it to a
+     * LangTag object consisting of a java.util.Locale object and the original
+     * String.
+     * @param langTagStr The language tag String from an xAPI Statement language map.
+     * @throws IllformedLocaleException when the String is not a valid language tag.
+     */
+    @JsonCreator
+    public LangTag(String langTagStr) throws IllformedLocaleException {
+        Builder builder = new Builder();
+        languageTagLocale = builder.setLanguageTag(langTagStr).build();
```

Comment:
there's a static helper for this. Builder doesnt even need to be imported.

https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/Locale.html#forLanguageTag(java.lang.String)

Reply:
https://github.com/yetanalytics/java-xapi-tools/pull/6/commits/791c4bfd97ee64ee342f78e7171539c6ed105ea5

Reply:
Ah now I know why I used the Builder class instead of forLanguageTag. The former throws an exception on invalid language tags; the latter doesn't, and that is why CI is failing now.